services/tilesets.js

```diff
@@ -16,6 +16,11 @@ var Tilesets = {};
  *
  * @param {Object} [config]
  * @param {string} [config.ownerId]
+ * @param {Object} [query]
+ * @param {number} [query.range]
```

Comment:
As far as I can see `range` is not defined https://docs.mapbox.com/api/maps/#list-tilesets